webapp/package.json

```diff
@@ -7,11 +7,10 @@
     "@fortawesome/free-brands-svg-icons": "^5.3.1",
     "@fortawesome/free-solid-svg-icons": "^5.3.1",
     "@fortawesome/react-fontawesome": "^0.1.3",
-    "@material-ui/core": "^3.1.2",
+    "@material-ui/core": "^3.0.3",
```

Comment:
why was this file touched?

Reply:
I'm not sure why. But I can revert those changes.

Reply:
yeah please revert the changes in this file.